webapp/src/test/resources/application-test.yml

```
@@ -9,4 +9,9 @@ permissionservice:
   endpoint:
 
 application:
-  state: CA
\ No newline at end of file
+  state: CA
+
+app:
+  iris:
+    vendorId: vendorId
+    url: http://my-iris.somewhere.org/iris/
```

It will only "fail fast" if you navigate to the right section in the UI.